### DISCUSSION TITLE
Credentials for geoserver's login

### DISCUSSION THREAD

**Discussion by itakartor** [kartoza/docker-geoserver#387] [Q&A]
Hi, i try to link postgis to geoserver with section on git, where i can use 2 docker container for use all together.
But i can't find credential for to login in manager. 
i try:
-admin , geoserver
-GeoServer , pass in container's log of geoserver
-admin , admin.

Someone can help me? 
<img width="440" alt="Immagine 2022-03-18 174854" src="https://user-images.githubusercontent.com/58552414/159046852-3a35307e-6c15-4a25-918f-387402f9d745.png">

**Comment by NyakudyaA** [comment]
> Hi, i try to link postgis to geoserver with section on git, where i can use 2 docker container for use all together. But i can't find credential for to login in manager. i try: -admin , geoserver -GeoServer , pass in container's log of geoserver -admin , admin.
> 
> Someone can help me? <img alt="Immagine 2022-03-18 174854" width="440" src="https://user-images.githubusercontent.com/58552414/159046852-3a35307e-6c15-4a25-918f-387402f9d745.png">

Can you please rather post what commands your are trying to use. Also I would encourage you to use the docker-compose provided as examples in the repo.

lastly please always be explicit when setting up containers by passing all env variables i.e 
```
GEOSERVER_ADMIN_PASSWORD=example
GEOSERVER_ADMIN_USER=example
```
then you will be able to access the container using the credentials `example:example`

**Reply by itakartor** [reply]
This is the commands:
docker run -d -p 5432:5432 --name db kartoza/postgis:13.0
docker run -d -p 8600:8080 --name geoserver --link db:db -e DB_BACKEND=POSTGRES -e HOST=db -e POSTGRES_PORT=5432 -e POSTGRES_DB=gis -e POSTGRES_USER=docker -e POSTGRES_PASS=docker kartoza/geoserver:2.18.0


I try docker-compose, but when i try to add postGis in geoserver there was other problem. Infact, geoserver call exeption for Connection refuse.

**Reply by NyakudyaA** [reply]
Doesn't https://github.com/kartoza/docker-geoserver/issues/397
Help you resolve the issue

**Reply by marsteen** [reply]
You have to provide user/password as environment-variable to the docker-container. Like this:

docker run -dit -p 8080:8080 --name geoserver -e "GEOSERVER_ADMIN_PASSWORD=geoserver" -e "GEOSERVER_ADMIN_USER=admin" kartoza/geoserver

Then I could login as user "admin" with password "geoserver"

**Reply by NyakudyaA** [reply]
This is now working correctly for a couple of versions now
